src/test/groovy/CategoryTest.groovy

```diff
@@ -68,8 +68,21 @@ final class CategoryTest extends GroovyTestCase {
         }
     }
 
-    // GROOVY-5245
+    // GROOVY-10133
     void testCategoryDefinedProperties2() {
+        assertScript '''
+            class Cat {
+                static boolean isAbc(self) { true }
+                static boolean getAbc(self) { true }
```

Comment:
don't you want false here?

Reply:
Fixed, thanks.  I also tested with the methods defined in the reverse order to be sure that wasn't a factor.